README.rst

```diff
@@ -429,10 +429,6 @@ Example result response (cut to one element only instead of 10):
 Filters
 -------
 
-**note**: this feature is compatible with django-filter up to version 0.15.
```

Comment:
Heh.. the reason for 1.0.0 is exactly to become stable - all the deprecations have been dropped.